virtcontainers/kata_agent.go

```diff
@@ -843,6 +848,43 @@ func (k *kataAgent) handleBlockVolumes(c *Container) []*grpc.Storage {
 	return volumeStorages
 }
 
+func (k *kataAgent) handlePidNamespace(grpcSpec *grpc.Spec, sandbox *Sandbox) (bool, error) {
+	sharedPidNs := false
+
+	for i, ns := range grpcSpec.Linux.Namespaces {
+		if ns.Type == string(specs.PIDNamespace) {
+			if ns.Path != "" {
+				if sandbox.state.Pid > 0 {
+					pidNsPath := fmt.Sprintf("/proc/%d/ns/pid", sandbox.state.Pid)
+
+					//  Check if pid namespace path is the same as the sandbox
+					if ns.Path == pidNsPath {
+						sharedPidNs = true
+					} else {
+						// Check for symlinks
```

Comment:
I'd remove this comment as it's obvious from the function call what it's doing.

Reply:
done